classes/class-aws-sns-topic.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * CCS Configuration.
+ *
+ * @package    WordPress
+ * @subpackage The Sun Theme
+ */
+
+use Aws\Sns\SnsClient;
+
+class TheSun_SSA_Base_Config {
+
+	protected $sns;
+
+	protected $is_feature_enabled;
+
+	protected $config;
+
+	protected $environment;
+
+	protected $topic_arn;
+
+	public function __construct() {
+		require AWS_PLUGIN_BASE_DIR . '/config/aws-config.php';
+		$this->config = $config;
+		$this->is_feature_enabled = get_option( 'thesun_disable_ccs_v3', 0 );
```

Comment:
Use Filter here
Example : https://github.com/arunchaitanyajami/aws-cloudwatch/blob/feature/sns-integration/classes/class-aws-cloud-watch.php#L56

apply_filters( 'aws_is_feature_enabled', false );

So users can use this filter, in the theme code to stop/start this feature
` add_filter('aws_is_feature_enabled', function( $value ){
   return get_option( 'thesun_disable_ccs_v3', 0 )
}, 10 ); `